modules/workflow-workflowoperation/src/main/java/org/opencastproject/workflow/handler/workflow/DuplicateEventWorkflowOperationHandler.java

```diff
@@ -229,11 +301,21 @@ public WorkflowOperationResult start(final WorkflowInstance workflowInstance, fi
       MediaPackage newMp = null;
 
       try {
+        String newMpId = workflowInstance.getConfiguration("newMpId");
+        if (newMpId == null) {
+          newMpId = UUID.randomUUID().toString();
+        }
         // Clone the media package (without its elements)
-        newMp = copyMediaPackage(mediaPackage, i + 1, copyNumberPrefix);
+        newMp = copyMediaPackage(mediaPackage, series, newMpId, noSuffix, i + 1, copyNumberPrefix);
+
+        if (series != null) {
+          newMp = ingestService
+                  .addCatalog(new ByteArrayInputStream(series.dc.toXmlString().getBytes(StandardCharsets.UTF_8)),
+                          UUID.randomUUID().toString() + ".xml", MediaPackageElements.SERIES, newMp);
+        }
 
```

Comment:
How is the new series ACL associated to this mediapackage?

Reply:
It...isn't. :D 
Again, good catch. I added code to copy the series ACL now.